[CI] ClientYamlTestSuiteIT test {yaml=search.highlight/70_number_of_fragments/number_of_fragments at the maximum should SUCCEED} failing

**Build Scans:**
- [elasticsearch-intake #45340 / rest-compat](https://gradle-enterprise.elastic.co/s/suqdgcjxshgiq)
- [elasticsearch-pull-request #169555 / rest-compatibility](https://gradle-enterprise.elastic.co/s/qekt7zjzjtrhq)
- [elasticsearch-pull-request #169550 / rest-compatibility](https://gradle-enterprise.elastic.co/s/ctbqanbebkhz4)
- [elasticsearch-pull-request #169550 / rest-compatibility](https://gradle-enterprise.elastic.co/s/klr4kicrd6gv6)
- [elasticsearch-pull-request #169543 / rest-compatibility](https://gradle-enterprise.elastic.co/s/6dyjgtw6tqgfg)
- [elasticsearch-pull-request #169543 / rest-compatibility](https://gradle-enterprise.elastic.co/s/plvleba5homvs)
- [elasticsearch-pull-request #169543 / rest-compatibility](https://gradle-enterprise.elastic.co/s/v4vu6y6sojp42)
- [elasticsearch-pull-request #169540 / rest-compatibility](https://gradle-enterprise.elastic.co/s/s5gtj6bznxpa2)
- [elasticsearch-pull-request #169539 / rest-compatibility](https://gradle-enterprise.elastic.co/s/ngp4ilfvgly7u)
- [elasticsearch-pull-request #169538 / rest-compatibility](https://gradle-enterprise.elastic.co/s/leg63q3mamqg2)

**Reproduction Line:**
```
./gradlew ":rest-api-spec:yamlRestCompatTest" --tests "org.elasticsearch.test.rest.ClientYamlTestSuiteIT" -Dtests.method="test {yaml=search.highlight/70_number_of_fragments/number_of_fragments at the maximum should SUCCEED}" -Dtests.seed=1975B7AA8CE02471 -Dtests.locale=vmw-MZ -Dtests.timezone=Egypt -Druntime.java=26
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.test.rest.ClientYamlTestSuiteIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('test%20%7Byaml%3Dsearch.highlight%2F70_number_of_fragments%2Fnumber_of_fragments%20at%20the%20maximum%20should%20SUCCEED%7D'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: org.junit.internal.runners.model.MultipleFailureException: There were 2 errors:
  java.lang.AssertionError(expected [2xx] status code but api [search] returned [400 Bad Request] [{"error":{"root_cause":[{"type":"x_content_parse_exception","reason":"[1:37] [highlight] failed to parse field [number_of_fragments]","stack_trace":"org.elasticsearch.ElasticsearchException$1: [1:37] [highlight] failed to parse field [number_of_fragments]\n\tat org.elasticsearch.server@9.6.0-SNAPSHOT/org.elasticsearch.ElasticsearchException.guessRootCauses(ElasticsearchException.java:812)\n\tat org.elasticsearch.server@9.6.0-SNAPSHOT/org.elasticsearch.ElasticsearchException.generateFailureXContent(ElasticsearchException.java:726)\n\tat org.elasticsearch.server@9.6.0-SNAPSHOT/org.elasticsearch.rest.RestResponse.build(RestResponse.java:202)\n\tat org.elasticsearch.server@9.6.0-SNAPSHOT/org.elasticsearch.rest.RestResponse.<init>(RestResponse.java:161)\n\tat org.elasticsearch.server@9.6.0
[truncated]
```

**Issue Reasons:**
- [main] 2 consecutive failures in test test {yaml=search.highlight/70_number_of_fragments/number_of_fragments at the maximum should SUCCEED}
- [main] 1 consecutive failures in step rest-compat
- [main] 14 consecutive failures in step rest-compatibility
- [main] 1 consecutive failures in pipeline elasticsearch-intake
- [main] 1 consecutive failures in pipeline elasticsearch-pull-request
- [main] 15 failures in test test {yaml=search.highlight/70_number_of_fragments/number_of_fragments at the maximum should SUCCEED} (40.5% fail rate in 37 executions)
- [main] 14 failures in step rest-compatibility (100.0% fail rate in 14 executions)
- [main] 7 failures in pipeline elasticsearch-pull-request (33.3% fail rate in 21 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-core-infra.

:eyes:

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -586,21 +586,6 @@ tests:
 - class: org.elasticsearch.xpack.esql.qa.single_node.EsqlSpecApproximationBucketIncludeEmptyBucketsIT
   method: test {csv-spec:bucket_include_empty_buckets.Large number of buckets doesn't OOM - stats}
   issue: https://github.com/elastic/elasticsearch/issues/155921
-- class: org.elasticsearch.test.rest.ClientYamlTestSuiteIT
-  method: test {yaml=search.highlight/70_number_of_fragments/number_of_fragments at the maximum should SUCCEED}
-  issue: https://github.com/elastic/elasticsearch/issues/155969
-- class: org.elasticsearch.test.rest.ClientYamlTestSuiteIT
-  method: test {yaml=search.highlight/70_number_of_fragments/negative number_of_fragments should FAIL}
-  issue: https://github.com/elastic/elasticsearch/issues/155970
-- class: org.elasticsearch.test.rest.ClientYamlTestSuiteIT
-  method: test {yaml=search.highlight/70_number_of_fragments/number_of_fragments above the maximum should FAIL}
-  issue: https://github.com/elastic/elasticsearch/issues/155972
-- class: org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT
-  method: test {p0=search.highlight/70_number_of_fragments/negative number_of_fragments should FAIL}
-  issue: https://github.com/elastic/elasticsearch/issues/155977
-- class: org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT
-  method: test {p0=search.highlight/70_number_of_fragments/number_of_fragments above the maximum should FAIL}
-  issue: https://github.com/elastic/elasticsearch/issues/155980
 - class: org.elasticsearch.xpack.stateless.reshard.StatelessReshardDisruptionIT
   method: testReshardWithDisruption
   issue: https://github.com/elastic/elasticsearch/issues/156004

